src/main/java/org/apache/sling/feature/cpconverter/handlers/slinginitialcontent/AssemblerProvider.java

```diff
@@ -99,9 +99,9 @@ private VaultPackageAssembler lazyConstruct(@NotNull BundleSlingInitialContentEx
     private void addPathFilterSetToAssemblerFilter(@NotNull PathEntry pathEntry, @NotNull VaultPackageAssembler assembler) {
         ImportMode importMode;
         if (pathEntry.isOverwrite()) {
-            importMode = ImportMode.UPDATE;
+            importMode = ImportMode.REPLACE;
```

Comment:
@niekraaijmakers , i had to lookup the definition of pathEntry.isOverwrite() which points to 
```
public abstract class ImportOptions {

	/**
	 * Specifies whether imported nodes should overwrite existing nodes.
	 * NOTE: this means the existing node will be deleted and a new node
	 * will be created in the same location.
	 * @return true to overwrite nodes, false otherwise
	 */
	public abstract boolean isOverwrite();
```
while i agree that ImportMode.REPLACE is probably the better match (see https://jackrabbit.apache.org/filevault/apidocs/org/apache/jackrabbit/vault/fs/api/ImportMode.html), i am wondering why we had UPDATE before. do you recall that?

i am bit concerned that changing it will cause regressions. in any case it would be great to have some dedicated tests for that illustrating the difference in behavior.

specially since there is probably not a 1:1 match with filevault.... e.g. there is also PathEntry.isPropertyOverwrite and isOverwrite is defined to only affect nodes.... while i believe that the fv-REPLACE affects both nodes and props, no?